src/core/entity.js

```diff
@@ -124,15 +125,45 @@ class Entity {
     return out;
   }
 
+  look_at(target_position) {
+    // Find the direction we're looking at.
+    const forward = zero_vector.slice();
+    math.vec3.subtract(forward, target_position, this.position);
+    math.vec3.normalize(forward, forward);
+
+    // Assume that the world's horizontal plane is the frame of reference for
+    // the look_at rotations. This should be fine for most game cameras which
+    // don't need to roll.
+
+    // Find left by projecting forward onto the world's horizontal plane and
+    // rotating it 90 degress counter-clockwise.
+    const left = math.vec3.from_values(forward[0], 0, forward[2]);
+    const rot = math.mat4.create();
+    math.mat4.rotate(rot, rot, Math.PI/2, [0, 1, 0]);
```

Comment:
Since this is a constant rotation, I think we could just hard-code the rotation matrix here in a future PR.